src/domain/createWorkspace.go

```diff
@@ -0,0 +1,30 @@
+package domain
+
+import (
+	"context"
+
+	"github.com/dennypenta/vel"
+)
+
+type CreateWorkspaceRequest struct {
+	UserID        string `json:"userID"`
```

Comment:
it should not be in the request body.
instead at the beginning of CreateWorkspace add profile getter

```go
	profile, rpcErr := h.GetProfile(ctx, struct{}{})
	if rpcErr != nil {
		return ConnectBranchResponse{}, rpcErr
	}

```

then get it using `profile.UserInfo.ID`